CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
+**January 3, 2024 [DESCW-1807](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1807)**
```

Comment:
one could argue we could be a bit more optimized for the changelog

Reply:
yeah i think it's time we had a chat as a team about the Changelog.

Adam was saying we might not need it for this project, though that would necessitate documenting our changes in some other way (perhaps more detailed/accurate commits?)

cya tomorrow